devicetypes/smartthings/zigbee-valve.src/zigbee-valve.groovy

```diff
@@ -11,100 +11,138 @@
  *  for the specific language governing permissions and limitations under the License.
  *
  */
-/*
- * Capabilities
- * - Battery
- * - Configuration
- * - Refresh
- * - Switch
- * - Valve
-*/
 
 metadata {
-	definition (name: "Zigbee Valve", namespace: "smartthings", author: "SmartThings") {
-		capability "Battery"
-		capability "Configuration"
-		capability "Refresh"
-		capability "Switch"
-		capability "Valve"
+    definition (name: "ZigBee Valve", namespace: "smartthings", author: "SmartThings") {
+        capability "Actuator"
+        capability "Battery"
+        capability "Configuration"
+        capability "Refresh"
+        capability "Valve"
 
-		fingerprint profileId: "0104", inClusters: "0000,0001,0003,0004,0005,0020,0006,0B02", outClusters: "0003"
-	}
+        attribute "powerType", "string"
```

Comment:
I think `powerSource` might be a little more clear of a name for this value. That also matches the name of the attribute in the ZCL spec. We should consider making this a standard capability as well, although i think that needs more discussion.